src/widget/wnumberpos.h

```diff
@@ -33,4 +35,7 @@ class WNumberPos : public WNumber {
     ControlProxy* m_pTimeRemaining;
     ControlProxy* m_pShowTrackTimeRemaining;
     ControlProxy* m_pTimeFormat;
+
+    ControlProxy* m_pBeatCountNextCue;
+    double pBeatsToNextCue;
```

Comment:
Try to get rid of this extra member. I think you can use the m_pBeatCountNextCue->get() instead.

Reply:
Been going for a while on this but I get an access exception when mixxx is loading. Not sure how the initialization sequence is, but it looks like I am trying to get the CP value, maybe even before the CO is registered.

What would be the best option to get the value from a CP directly? (ie not in a slot that is connected to the value changed signal)

![1](https://user-images.githubusercontent.com/4730927/173234408-d0f4ef5a-9a7d-48b9-af86-3b72cb6b22b9.png)
![image_2022-06-12_145843918](https://user-images.githubusercontent.com/4730927/173234411-116a86a1-5538-4345-9de6-6bb216dfb8fc.png)

Reply:
The code of this crasher is gone due to the rebase.  

But it looks like you try to access an uninitialized pointer. Pointers are not initialized and hold whatever was at that memory location before.  This can be fixed by using our parented_ptr that we hav introduced to indicate that the ownership has been passed to the parent object and it is not necessary to delete it manually. 
  
`parented_ptr<ControlProxy> m_pBeatCountNextCue;`

Reply:
Thanks ! That did the trick, I just removed the unnecessary member and it is working as you said. Tks !